src/mongo/delegates/SearchDocuments.class.php

```diff
@@ -330,7 +330,7 @@ private function addValuesToTarget($values, $field, &$target)
         
 		if(count($values) > 0){						
 	        for ($i=0; $i<$limit; $i++) {	        	
-	            $v = utf8_encode($values[$i]); //todo: added by CJC
```

Comment:
Agreed.  Although utf8_encode will only work if the input is ISO-8859-1, any other encoding and it will result in strange characters, so I can't understand the rationale behind adding this here.  There doesn't seem to be any other part of tripod - as far as I can see - which is attempting to ensure things are encoded in utf-8, and doing so is very difficult without knowing the source encoding, so I think it's simply down to the calling app to make sure that UTF-8 is passed in.